lib/workers/bin/run_single_worker.rb

```diff
@@ -59,6 +59,12 @@
 require File.expand_path("../../../config/environment", __dir__)
 
 worker_class = worker_class.constantize
+missing_roles = ServerRole.where(:name => worker_class.required_roles) - MiqServer.my_server.active_roles
+unless missing_roles.empty?
+  puts "ERR:  Server roles are not sufficient for `#{worker_class}` worker. Missing: #{missing_roles.collect(&:name)}"
+  exit 1
```

Comment:
@tumido @carbonin do we really want to `exit 1`?  This workflow is to help developers, not break containers or those launching from `foreman start`.  Will this non-zero RC propagate up so someone running the pods will be able to know what happened?  I don't think it should ever happen but maybe?

Reply:
In the `foreman start` case, I think this would break, but I consider that more close to a developer usecase anyway. It shouldn't ever happen in containers because the server is starting the worker rather than a user.

Reply:
Can we possibly add a `-f/--force` option to opt-out of the `exit 1`?  This way, in a non-prod environment, we can just spin up the code for this and only the warning is shown?  Something like:

```ruby
exit 1 unless options[:force]
```